n9kv/Makefile

```diff
@@ -1,17 +1,12 @@
 VENDOR=Cisco
-NAME=NXOS 9000v
+NAME=n9kv
 IMAGE_FORMAT=qcow2
 IMAGE_GLOB=*.qcow2
 
-# Match versions similar to the following:
-# - nxosv-final.7.0.3.I7.5a.qcow2
-# - nxosv-final.7.0.3.I7.9.qcow2
-# - nxosv.9.2.1.qcow2
-# - nxosv.9.2.4.qcow2
-# - nexus9300v.9.3.9.qcow2
-# - nexus9300v.9.3.10.qcow2
-# - nexus9300v64.10.2.2.F.qcow
-VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+\?\.\(\(7\.0\.3\.I[0-9]\.[0-9a-z]\+\)\|\([0-9]\+\.[0-9]\+\.[0-9]\+\)\)\(\..*\|$$\)/\1/')
+# rename the disk image file as n9kv-<version>.qcow2
+# examples:
+# for a file named "n9kv_9300-10.5.2.qcow2" the image will be named "vrnetlab/cisco_n9kv:9300-10.5.2"
```

Comment:
@hellt FYI, this comment is incorrect I think. The `_9300` bit breaks the version matching. For me this is only working when the image is `n9kv-10.5.2.qcow2`

Reply:
yes, fixed the comment.
name the as `n9kv-9300-15.3.2.qcow2` and it will be fine. Basically just typed `_` instead of `-`